charts/kube-prometheus-stack/Chart.yaml

```diff
@@ -0,0 +1,42 @@
+annotations:
+  artifacthub.io/links: |
+    - name: Chart Source
+      url: https://github.com/prometheus-community/helm-charts
+    - name: Upstream Project
+      url: https://github.com/prometheus-operator/kube-prometheus
+  artifacthub.io/operator: "true"
+apiVersion: v2
+appVersion: 0.45.0
+dependencies:
+- condition: nodeExporter.enabled
+  name: prometheus-node-exporter
+  repository: https://prometheus-community.github.io/helm-charts
```

Comment:
The repository and version in dependencies is only used in case the charts are not present locally and need to be downloaded - https://helm.sh/docs/helm/helm_dependency/
Also the charts are not updated automatically, there is a separate command for that -  https://helm.sh/docs/helm/helm_dependency_update/